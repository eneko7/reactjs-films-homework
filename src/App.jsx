import React from 'react';
import Footer from './components/Footer/Footer.jsx';


class App extends React.Component {
    render() {
        return(
            <div className="main_container">
                <Footer />
            </div>
        );
    }
};

export default App; 