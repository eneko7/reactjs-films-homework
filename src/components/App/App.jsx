import React from 'react';
import './App.scss';
import Footer from '../Footer/Footer.jsx';


class App extends React.Component {
    render() {
        return(
            <div className="main_container">
                <Footer />
            </div>
        );
    }
};

export default App; 