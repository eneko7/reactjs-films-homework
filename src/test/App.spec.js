import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('App', () => {
    it('App renders correctly', () => {
        const renderedApp = renderer.create(<App />).toJSON();
        expect(renderedApp).toMatchSnapshot();
    });
});