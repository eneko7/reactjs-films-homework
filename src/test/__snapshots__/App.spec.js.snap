// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders correctly 1`] = `
<div
  className="main_container"
>
  <footer
    className="main_container_footer"
  >
    <span
      className="main_container_footer_films"
    >
      FILMS
    </span>
    <span
      className="main_container_footer_copyright"
    >
      Copyright Â© 
      2018
       FILMS. 
      <span
        className="main_container_footer_copyright_signature"
      >
        IHAR KARPUK
      </span>
    </span>
  </footer>
</div>
`;
